static/app/views/performance/newTraceDetails/traceDrawer/details/issues/issues.tsx

```diff
@@ -201,6 +201,7 @@ function IssueListHeader({node}: {node: TraceTreeNode<TraceTree.NodeValue>}) {
                       start: dateSelection.start,
                       end: dateSelection.end,
                       statsPeriod: dateSelection.statsPeriod,
+                      project: -1,
```

Comment:
Lets add a comment why we need this and what happens if we dont add it. It looks innocent, but seems like all hell breaks loose if we dont do this :D